acquia/acquia_cms
# ACMS-1542: Support for Site Studio 7.0.x

**Motivation**

Fixes #ACMS-1542

**Proposed changes**

We are re-exporting site studio packages as per 7.x


**Alternatives considered**


**Testing steps**


- Checkout to ACMS-1542
- Run composer require drupal/core:^9.5
- Site install with site studio key present ./vendor/bin/acms si
- Update text format site studio to use ckeditor5
- Create page with any component using text editor and make sure its working as expected.


**Merge requirements**
